solver/src/main/java/org/chocosolver/solver/objective/IBoundsManager.java

```diff
@@ -46,27 +46,29 @@ default boolean isOptimization() {
      * States that lb is a global lower bound on the problem
      *
      * @param lb lower bound
+     * @return true iff the lower bound has been updated
      */
-    void updateBestLB(Number lb);
+    boolean updateBestLB(Number lb);
 
     /**
      * States that ub is a global upper bound on the problem
      *
      * @param ub upper bound
+     * @return true iff the upper bound has been updated
      */
-    void updateBestUB(Number ub);
+    boolean updateBestUB(Number ub);
 
-    /**
-     * States that b is global bounds on the problem
-     *
-     * @param b lower and upper bounds
-     */
-    default void updateBestBounds(IBoundsManager b) {
-        if(isOptimization()) {
-            updateBestLB(b.getBestLB());
-            updateBestUB(b.getBestUB());
-        }
-    }
+    //    /**
```

Comment:
Why is this part commented?

Reply:
It was not used anywhere in the code according to IntelliJ so I commented it to see if everything was still working locally and I forgot to delete this dead code.

Reply:
I think this is related to [eps4j-mpj-choco](https://github.com/eps4j/eps4j-mpj-choco), but the repo is not active anymore, so let's remove it.